.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+  "extends": "plugin:@tinymce/standard",
+  "parserOptions": {
+    "sourceType": "module",
+    "project": [
+      "./tsconfig.json",
+      "modules/*/tsconfig.json"
```

Comment:
eslint doesn't support project references? interesting 🤔

Reply:
Unless something has changed recently:
https://github.com/typescript-eslint/typescript-eslint/issues/856

Reply:
well that sucks, it seems like it's by design

>  in particular, the goal of project references is that when project a references project b, project a does not include the .ts files in project b.